packages/storage/src/core/uploaders/ipfs-uploader.ts

```diff
@@ -301,8 +311,16 @@ export class IpfsUploader implements IStorageUploader<IpfsUploadBatchOptions> {
       );
     }
 
+    if ((this.authToken && this.clientId) || (this.authToken && this.secretKey)) {
+      throw new Error(
+        "Cannot use both auth token and client ID or secret key. Please use auth token or secret key for server-side applications or clientId for client-side applications.",
+      );
+    }
```

Comment:
this feels wrong

Reply:
auth token = either server or client
clientId = exclusively client
secretKey = exclusively server

Reply:
I thought we were saying that the auth token should be treated like a secret, so we shouldn't allow it client side?